### PR TITLE
New parallel execution

### PR DESCRIPTION
**Pull request by Jarlaxle91**

### CODE REVIEW

**Comment by erdi** [comment]
There’s quite a lot of change in this PR and no explanation/context at all in the description. Can you please explain what this is about and why you believe it’s needed/valuable?

File: build.gradle [hunk 1]
```diff
@@ -1,71 +1,28 @@
...
-        group JavaBasePlugin.VERIFICATION_GROUP
-
-        outputs.upToDateWhen { false }  // Always run tests
-
-        systemProperty "geb.build.reportsDir", reporting.file("geb/$name")
-        systemProperty "geb.env", driver
-    }
+    testImplementation 'io.github.bonigarcia:selenium-jupiter:4.0.1'
+    testImplementation 'org.seleniumhq.selenium:selenium-java:4.1.2'
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
```
Comment (achimgrimm) [inline build.gradle:13]: This does change to project over to junit5. The readme needs to be updated as well.

File: build.gradle [hunk 2]
```diff
@@ -1,71 +1,28 @@
...
+        events "passed", "skipped", "failed"
     }
-}
 
-tasks.withType(GroovyCompile) {
-    groovyOptions.forkOptions.memoryMaximumSize = '256m'
+    systemProperty("junit.jupiter.execution.parallel.enabled" , "true")
+    systemProperty("junit.jupiter.execution.parallel.config.strategy", "fixed")
+    systemProperty("junit.jupiter.execution.parallel.mode.default", "concurrent")
+    systemProperty("junit.jupiter.execution.parallel.config.fixed.parallelism", 2)
```
Comment (achimgrimm) [inline build.gradle:27]: Will the parallel execution work with the spoc test runner?

File: src/test/groovy/GebishOrgSpec.groovy [hunk 3]
```diff
@@ -1,21 +0,0 @@
-import geb.spock.GebSpec
-
-class GebishOrgSpec extends GebSpec {
```
Comment (achimgrimm) [inline src/test/groovy/GebishOrgSpec.groovy:3]: This is the only spock test - You need to keep at least one test to verify that the spock integration stll works.

**Pull request closed without merging**